lib/tasks/level_up_invetory_in.rake

```diff
@@ -0,0 +1,17 @@
+namespace :kit_allocation do
+  desc "Level up inventory in to inventory out in kit_allocations"
+  task inventory_in: :environment do |task|
+    KitAllocation.where(kit_allocation_type: "inventory_in").destroy_all
+    out_kit_allocations = KitAllocation.where(kit_allocation_type: "inventory_out")
+    out_kit_allocations.each do |out_kit_allocation|
+      in_kit_allocation = KitAllocation.create!(storage_location_id: out_kit_allocation.storage_location_id,
+        organization_id: out_kit_allocation.organization_id,
+        kit_id: out_kit_allocation.kit_id,
+        kit_allocation_type: "inventory_in")
+      out_kit_allocation.line_items.each do |line_item|
+        in_kit_allocation.line_items.create!(item_id: line_item.item_id, quantity: line_item.quantity * -1)
```

Comment:
@cielf this _seems_ like it'd work, but it takes the "out allocations" as the source of truth and redoes the "in allocations" to mirror them. Have we validated that that's the best course of action here?

Reply:
@dorner, @zeeshan-haidar   I'm just looking at this in isolation -- and assuming there aren't a lot of side effects.    

This would set the inventory in to the inventory out, right?  It does makes sense to me  that the inventory in would account for the inventory out.  *But* that would not take into account any existing inventory.   So if you have distributed 1000 kits, but you have 100 on hand,  your inventory in should reflect the 1100 kits,  not just the 1000.

Reply:
@cielf if 1000 distributed kits what I understood are the one which our current inventory in query is showing, and Inventory in in-hand what I understood are the Allocated Kits, then in that case yes our inventory in query will show 1100 kits, since this change is including kits allocated to the inventory in query

Reply:
That sounds ok to me, then.